cmd/core-command/Dockerfile

```diff
@@ -26,7 +26,7 @@ WORKDIR /edgex-go
 
 RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
 
-RUN apk add --update --no-cache make git
+RUN apk add --update --no-cache make git zeromq-dev libsodium-dev pkgconfig build-base
```

Comment:
It is more than just the packages in the build stage. Also need ZMQ in the resulting image in each of these docker files.

https://github.com/edgexfoundry/edgex-go/blob/main/cmd/core-data/Dockerfile#L52

Reply:
added